packages/test/src/workflows/cancel-fake-progress.ts

```diff
@@ -40,3 +45,8 @@ export const cancelFakeProgress: ActivitySignalHandler = () => {
     },
   };
 };
+
+export const interceptors = (): WorkflowInterceptors => ({
+  inbound: [new OpenTelemetryInboundInterceptor()],
+  outbound: [new OpenTelemetryOutboundInterceptor()],
+});
```

Comment:
This doesn't belong here, it has nothing to do with this WF.

Reply:
Where should it go then? I think this is something we could be more clear on or make more obvious in the API. Having to load up interceptors with the workflows makes sense when they affect them but it's not clear where they're expected to go if not